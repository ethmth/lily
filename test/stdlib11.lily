
let a::int = range(4, 9)

a.remove(6)

a.reverse()

a.print()