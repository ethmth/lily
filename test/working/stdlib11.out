[8, 7, 5, 4]
